### ISSUE TITLE
Auto deploy not working?

### ISSUE THREAD

**Issue by cskiwi** [kubero-dev/kubero#306]
Hi, 

I've setup a app with the autodeploy enabled, however when I push new commits nothing happens.
![image](https://github.com/kubero-dev/kubero/assets/847540/cdd52161-34fa-4d3b-a22c-0ca97d94c88d)

Is there any configuration I have to do? or are there any specific scopes that need to be defined for may acces token?
because I used the onces defined here: [documentation](https://docs.kubero.dev/Installation/repositories)

**Comment by mms-gianni** [comment]
Make sure your Pipeline is connected. It works only on connected repositories, since this process installs the webhook and creates the deployment keys. 

You should see the webhooks github is sending here: https://github.com/cskiwi/yourRepo/settings/hooks

**Comment by mms-gianni** [comment]
Oh and it's also important to use SSL on your webhook endpoint. I really need to mention this in the documentation somewhere.

**Comment by cskiwi** [comment]
Going to the github webhooks settings I get:
```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Error</title>
</head>

<body>
    <pre>Cannot POST /api/webhook/github</pre>
</body>

</html>
```

The webhook is: https://domain.com/api/webhook/github that should be correct right?

**Comment by mms-gianni** [comment]
No. It should be https://domain.com/repo/webhooks/github 

Where did this webhook URL come from? Kubero CLI?

**Comment by cskiwi** [comment]
> No. It should be https://domain.com/api/repo/webhooks/github
> 
> Where did this webhook URL come from? Kubero CLI?
> 
> //edit: added /api/ in the URL

Via the "Connect" button when creating a new pipeline

It also added `:80` to the url, with the addition of `/repo/webhooks` and removing the `:80` it works, Ill make a PR later today if not fixed by then :)

**Comment by mms-gianni** [comment]
Sure. Still reviewing all your previous PR's 😄 appreciate them.  

Wondering where the :80 came from then. 

I was investigating, why this path is not listed in the API docs. It seems Swaggergen is ignoring the ".all" method. 

https://github.com/kubero-dev/kubero/blob/main/server/src/routes/repo.ts#L47

**Comment by cskiwi** [comment]
I was looking into why the webhook was wrong, but I can't seem to find where it's set.
It is using the environment variable: `KUBERO_WEBHOOK_URL`

But I can't seem to remember setting it, and i find nothing in the [code](https://github.com/search?q=org%3Akubero-dev%20KUBERO_WEBHOOK_URL&type=code) that would define this.

**Comment by mms-gianni** [comment]
Interesting. 

My first doubt was the kubero cli was proposing the wrong URL. But seems OK. 

https://github.com/kubero-dev/kubero-cli/blob/8e7002d25ff1764f931f8e8131a9860029d76e8a/cmd/kuberoCli/install.go#L575

I plan to separate the webhook URL and the rest of the UI.

**Comment by mms-gianni** [comment]
Found the config and fixed in version v0.0.158 of the operator

https://github.com/kubero-dev/kubero-operator/blob/v0.0.157/config/samples/application_v1alpha1_kubero.yaml#L59
